pepper-apis/studybuilder-cli/studies/singular/patch-log.conf

```diff
@@ -3,7 +3,6 @@
     "SingularPatientSurveyUpdate",
     "SingularUpdates",
     "SingularAboutHealthyPexTask",
-    "SingularUpdateConsentCopyEvents",
     "SingularMakeNoMedicalTreatmentOptionExclusive",
     "SingularUpdateConsentCopyEvents"
     "SingularFixRaceQuestionTypo"
```

Comment:
Not directly part of PR objective, but there was an entry in patch-log.conf that was duplicated and messing up applying the patches.